lib/bolt/config/transport/local.rb

```diff
@@ -10,6 +10,7 @@ class Local < Base
         WINDOWS_OPTIONS = %w[
           bundled-ruby
           cleanup
+          extensions
```

Comment:
This makes `extensions` available when running on a *nix controller, it just won't do anything. We _could_ limit this to just Windows controllers, but if the project is run on both Windows and *nix then an "unknown config option" warning would show up when run on *nix.

Reply:
I think this is fine.